src/limits.hpp

```diff
@@ -117,7 +117,8 @@ integerConversion( INPUT input )
   static_assert( std::is_integral< INPUT >::value, "INPUT must be an integral type." );
   static_assert( std::is_integral< OUTPUT >::value, "OUTPUT must be an integral type." );
 
-  return OUTPUT{ input };
+//  return OUTPUT{ input };
+  return static_cast<OUTPUT>(input);
```

Comment:
The braced initialization broke something?

Reply:
yes...but I think I fixed it elsewhere. I will try to revert this.

Reply:
there was an integer narrowing error.